expanding on TrustStore and KeyStore #4578 #4060 (#7015)
opensearch-project/documentation-website: _security/configuration/generate-certificates.md

rm client.ext
```

## Sample script to convert PEM certificates to keystore and truststore

> If you want to generate keystore and truststore from the previously generated PEM certificates, the following script can be used:

Reviewer: ```suggestion
If you want to generate keystore and truststore from the previously generated PEM certificates, use the following script:
```

> If you want to generate a keystore and a truststore from the previously generated PEM certificates, then use the following script:

Reviewer 2: ```suggestion
You can use the following script to generate a keystore and a truststore from the previously generated PEM certificates:
```
